provisioner/k8s/cmd/run.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
+
+	k8scontroller "github.com/operator-framework/rukpak/provisioner/k8s/controller"
+)
+
+func init() {
+	rootCmd.AddCommand(runCmd)
+}
+
+var runCmd = &cobra.Command{
+	Use:   "run",
+	Short: "run the provisioner",
```

Comment:
This doesn't have to be in this PR but can we update this command at some point to avoid printing the command usage when encountering an error (`SilenceUsage: true`).

Reply:
done